build-system/pr-check.js

```diff
@@ -326,6 +329,7 @@ function runAllCommands() {
     command.runVisualDiffTests(/* opt_mode */ 'master');
     command.runJsonAndLintChecks();
     command.runDepAndTypeChecks();
+    command.verifyVisualDiffTests();
```

Comment:
Don't you want to do this after `runUnitTests`?

Reply:
No, this was intentional, because by the time `runJsonAndLintChecks` and `runDepAndTypeChecks` are run, we can be sure that the visual build has been processed. This way, we can fail early if there are visual diffs, instead of waiting for the unit tests to run.

Let me know if you'd prefer that the check be moved to the end.

Reply:
Here's an example of the timing I describe above: https://travis-ci.org/ampproject/amphtml/jobs/273041116

Reply:
I think I'd prefer to do it later. I can see that we'll add so many v-diff tests that they will go much slower.

Reply:
SGTM. Just sent you #11211 with the change.